mac

```diff
@@ -166,7 +166,14 @@ brew_install_or_upgrade 'hub'
 # shellcheck disable=SC2016
 append_to_file "$HOME/.zshrc" 'eval "$(hub alias -s)"'
 
-brew_install_or_upgrade 'node'
+if ! command -v n > /dev/null; then
+  fancy_echo 'Installing n and latest Node.js and NPM...'
+  curl -L http://git.io/n-install | bash -s -- -y latest
+else
+  fancy_echo 'Updating n...'
+  n-update -y
+fi
+
```

Comment:
The `n-install` script actually does this for you. By default, everything is done with the assumption of `N_PREFIX=$HOME/n`, but we can change this as well if necessary (maybe use a path in `/usr/local` like brew?), by providing `N_PREFIX` during install.  See: https://github.com/mklement0/n-install#examples